patches/src/main/resources/youtube/settings/xml/revanced_prefs.xml

```diff
@@ -913,12 +913,12 @@
             </PreferenceCategory>
 
             <PreferenceCategory android:title="@string/revanced_preference_screen_misc_title" android:layout="@layout/revanced_settings_preferences_category">
+                <Preference android:title="Bypass link redirection" android:summary="@string/revanced_patches_excluded" android:selectable="false"/>
```

Comment:
It seems that [URL redirection](https://en.wikipedia.org/wiki/URL_redirection) is more commonly used than `link redirection`.

It seems better to use `Bypass URL redirects`, the patch name adopted by ReVanced.